loopchain/peer/peer_service.py

```diff
@@ -281,4 +275,4 @@ def close(self):
             await StubCollection().create_icon_score_stub(channel_name)
 
     def _load_channel_infos(self):
-        self._channel_infos = self._get_channel_infos()
+        self._channel_infos = conf.CHANNEL_OPTION
```

Comment:
Is this method needed?
How about replace `self._load_channel_infos()` as `self._channel_infos = conf.CHANNEL_OPTION`
and remove this method?

Reply:
Good idea!